Server/source/Sockets.cpp

```diff
@@ -1,30 +1,16 @@
 #include "sockets.h"
-#include <string.h>
+#include "Results.h"
 #include <sys/socket.h>
 #include <arpa/inet.h>
-#include "Results.h"
-
-using namespace std;
-
-#define PORT 0xCAFE
+#include <cstring>
 
-int sendData(int sock, u64 tid, string name)
+int sendData(int sock, u64 tid, const char *name)
 {
-    //titlepacket *packet = new titlepacket();
     struct titlepacket packet;
```

Comment:
```suggestion
    titlepacket packet;
```
This is C++. No need to declare structs in C-style